### PR TITLE
cpdf: add page

### PR DESCRIPTION
**Pull request by andrik**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Hi guys, this one is from #3689 I'll try to create other ones. As the cheat project allows more examples then we do, I decided to pick up what looks like the most common ones.

### CODE REVIEW

File: pages/common/cpdf.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# cpdf
+
+> CLI to manipulate existing PDF files in a variety of ways.
+> Homepage: <https://www.coherentpdf.com/cpdfmanual/cpdfmanual.html>.
+
+- Select pages 1, 2, 3 and 6 from source document and write those to destination document:
```
Comment (einverne) [inline pages/common/cpdf.md]:
```suggestion
- Select pages 1, 2, 3 and 6 from source document and write those to a destination document:
```

File: pages/common/cpdf.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`cpdf -merge {{path/to/source_document_one.pdf}} {{path/to/source_document_two.pdf}} -o {{path/to/destination_document.pdf}}`
+
+- Show the bookmarks of a document:
+
+`cpdf -list-bookmarks {{path/to/document.pdf}}`
+
+- Split a document into ten-page chunks, writing them to chunk001.pdf, chunk002.pdf, etc:
+
+`cpdf -split {{path/to/document.pdf}} -o {{path/to/chunk%%%.pdf}} -chunk 10`
```
Comment (agnivade) [inline pages/common/cpdf.md]: 10 should be in tokens (`{{}}`)

File: pages/common/cpdf.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Encrypt a document using 128bit encryption, providing `fred` as owner password and `joe` as user password:
+
+`cpdf -encrypt 128bit fred joe {{path/to/source_document.pdf}} -o {{path/to/encrypted_document.pdf}}`
```
Comment (agnivade) [inline pages/common/cpdf.md]: 128bit, fred and joe should be in tokens (`{{}}`)

File: pages/common/cpdf.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- Decrypt a document using the owner password `fred`:
+
+`cpdf -decrypt {{path/to/encrypted_document.pdf}} owner=fred -o {{path/to/decrypted_document.pdf}}`
```
Comment (agnivade) [inline pages/common/cpdf.md]: fred should be in tokens (`{{}}`)

File: pages/common/cpdf.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Show the annotations of a document:
+
+`cpdf -list-annotations {{path/to/document.pdf}}`
+
+- Create a new document, with metadata, based on an existing one:
```
Comment (agnivade) [inline pages/common/cpdf.md]: Let's change this to "Create a new document from an existing one with additional metadata"

File: pages/common/cpdf.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
+# cpdf
+
+> CLI to manipulate existing PDF files in a variety of ways.
+> Homepage: <https://www.coherentpdf.com/cpdfmanual/cpdfmanual.html>.
+
+- Select pages 1, 2, 3 and 6 from source document and write those to a destination document:
```
Comment (agnivade) [inline pages/common/cpdf.md]: from _a_ source document.

File: pages/common/cpdf.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+`cpdf {{path/to/source_document.pdf}} 1-3,6 -o {{path/to/destination_document.pdf}}`
```
Comment (agnivade) [inline pages/common/cpdf.md]: 1-3,6 should be in tokens (`{{}}`)

File: pages/common/cpdf.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
+# cpdf
+
+> CLI to manipulate existing PDF files in a variety of ways.
+> Homepage: <https://www.coherentpdf.com/cpdfmanual/cpdfmanual.html>.
```
Comment (ivanhercaz) [inline pages/common/cpdf.md]: After review it again to translate it to Spanish, I think the most consistent way to put an URL for the command is "More information". Although I haven't read a guideline or a rule about it, all the pages I remember to read have "More information". Should we unify the line that provides the URL?

Comment (agnivade) [reply pages/common/cpdf.md]: Yes, that's right. We should use "More information"

**Pull request merged**